pandas/tests/indexes/test_base.py

```diff
@@ -48,7 +48,8 @@ def setUp(self):
                             catIndex=tm.makeCategoricalIndex(100),
                             empty=Index([]),
                             tuples=MultiIndex.from_tuples(lzip(
-                                ['foo', 'bar', 'baz'], [1, 2, 3])))
+                                ['foo', 'bar', 'baz'], [1, 2, 3])),
+                            mixedIndex=Index([0, 'a', 1, 'b', 2, 'c']))
```

Comment:
you can just skip those in the acutal test running if need be (this is done for example with MultiIndex a bit, where you are looping thru the various tests objects but skipping for MultiIndex).